docs/security/trust/trust_delegation.md

```diff
@@ -0,0 +1,222 @@
+```
+
+The revocation will take effect as soon as you publish:
+
+```
+$ notary -d ~/.docker/trust publish docker.io/<username>/<imagename>
+```
+
+Note that by removing all the keys from the `targets/releases` delegation, the
+delegation (and any tags that are signed into it) is removed.  That means that
+these tags will all be deleted, and you may end up with older, legacy tags that
+were signed directly by the targets key.
+
+## Removing the `targets/releases` delegation entirely from a repository
+
+If you've decided that delegations aren't for you, you can delete the
+`targets/releases` delegation entirely. This also removes all the tags that
+are currently in `targets/releases`, however, and you may end up with older,
+legacy tags that were signed directly by the targets key.
+
+To delete the `targets/releases` delegation:
+
+```
+$ notary -d ~/.docker/trust delegation remove docker.io/<username>/<imagename> targets/releases
+
+Are you sure you want to remove all data for this delegation? (yes/no)
+yes
+
+Forced removal (including all keys and paths) of delegation role targets/releases to repository "docker.io/<username>/<imagename>" staged for next publish.
+
+$ notary -d ~/.docker/trust publish docker.io/<username>/<imagename>
+```
+
+## Pushing trusted data as a collaborator
+
+As a collaborator with a private key that has been added to a repository's
+`targets/releases` delegation, you need to import the private key that you
+generated into Content Trust.
+
+To do so, you can run:
+
+```
+$ notary -d ~/.docker/trust key import delegation.key --role user
+```
+
+where `delegation.key` is the file containing your PEM-encoded private key.
+
+After you have done so, running `docker push` on any repository that
+includes your key in the `targets/releases` delegation will automatically sign
+tags using this imported key.
+
+## `docker push` behavior
+
+When running `docker push` with Docker Content Trust, Docker Engine 1.10 and above
+will attempt to sign and push with the `targets/releases` delegation if it exists.
+If it does not, the targets key will be used to sign the tag, if the key is available.
+
+## `docker pull` behavior
+
+When running `docker pull` with Docker Content Trust on Docker Engine 1.10,
+Docker will attempt to search in the `targets/releases` delegation for the signed
+image tag, and will fall back to the legacy `targets` tags if it does not exist.
+
+This means that if you have an existing image repository with signed tags, and then
+add the `targets/releases` delegation as per the instructions above, the
+original tags signed with the targets keys will still be `docker pull`-able.
+
+Please note that when searching for signed tags, Docker 1.10 may pick
+up on other non-`targets/releases` delegations' signed tags if they exist for
+this image repository. In Docker 1.11, this behavior is removed such that all
```

Comment:
The description of "pre 1.11 and post 1.11" differences in this document make it hard to read, and may be better included in the changelog.

Also, if the 1.10 descriptions point to the "experimental" features, we should not mention them at all.